site/content/docs/main/file-system-backup.md

```diff
@@ -22,9 +24,8 @@ have a native snapshot concept, FSB might be for you.
 the one backing Kubernetes volumes, for example, a durable storage.
 
 Cons:
-- It backs up data from the live file system, so the backup data is less consistent than the snapshot approaches.
-- It access the file system from the mounted hostpath directory, so the pods need to run as root user and even under 
-privileged mode in some environments.  
+- It backs up data from the live file system, in which way the data is not captured at the same point in time, so is less consistent than the snapshot approaches.
```

Comment:
I think this scenario is also worth mentioning here.
Some k8s system not using the actual pod UUID in the kubelet pod directory also cannot use filesystem backup.
https://kubernetes.slack.com/archives/C6VCGP4MT/p1687321544079519

Reply:
Done, I added it in the Limitation section, as it is not a general issue.